---
title: "Ahmed Lab for Genomics and Artificial Intelligence"
author: "Firoz Ahmed, Ph.D."
date: "2023-07-15"
output:
  html_document:
    css: styles.css
    self_contained: yes
    keep_md: yes
    fig_caption: yes
    number_sections: no
    toc: yes
    toc_float: yes
    code_download: no
    code_external: yes
    code_folding: hide
    df_print: paged
    toc_depth: 3
    toc_unnumbered: no
    theme: cosmo
    highlight: tango
    lib_dir: output_folder
    code: FALSE
  pdf_document:
    toc_depth: '3'
    
---

# About {#about}
Greetings!

<div align="left">
  <img src="images/DrFiroz.jpeg" alt="Lab Research" width="100" height="100" style="float:left; margin-right:20px;">

<p style="text-align: justify;">
Thank you for visiting the __Ahmed Lab__ website. We greatly appreciate your interest in our work, where the powerful integration of multi-omics data, bioinformatics, and artificial intelligence (AI) drives us to revolutionize our understanding of life's enigmatic intricacies and complex diseases. Our primary objective is to identify early biomarkers and potential disease targets, paving the way for groundbreaking advancements in the field of personalized medicine.
</p>
</div>

# Research {#research}

Our lab conducts research in various areas, including:  



- __Exploring the Molecular Pathways of Abnormal Cell Proliferation in Lung Cancer__
<p style="text-align: justify;">
Our research is dedicated to unraveling the intricate mechanisms underlying non-small cell lung cancer (NSCLC) and identifying the key regulators responsible for uncontrolled cell growth within this disease. NSCLC is characterized by the disruption of normal cell division, but the precise molecular processes driving this phenomenon are not yet fully understood. Through the use of advanced computational techniques and in-depth analysis of gene expression and protein interactions, our aim is to uncover the master regulators and specific network of genes that play a pivotal role in driving cell proliferation in NSCLC. These findings hold great potential as valuable biomarkers and therapeutic targets for the development of more effective treatments. Ultimately, our goal is to improve outcomes and enhance the quality of life for individuals affected by this challenging condition.
</p>

- __Advancing Predictive Models for Early Cancer Diagnosis and Prognosis__
<p style="text-align: justify;">
Our lab leverages the capabilities of artificial intelligence (AI) and advanced data analytics to create predictive models for early cancer diagnosis and prognosis. By utilizing AI algorithms and machine learning techniques, we analyze and integrate diverse datasets, including multi-omics and clinical data, to uncover hidden patterns and molecular signatures associated with different types of cancer. This enables us to identify biomarkers and genetic markers that play a crucial role in accurate cancer detection and prognosis. Through our research, we strive to harness the potential of AI to advance personalized medicine and improve patient outcomes in the fight against cancer.
</p>

- __Investigating the Underlying Mechanisms of Diabetes Pathogenesis__  
<p style="text-align: justify;">
Diabetes, a complex disease that impacts millions of people globally, remains a significant health concern. At our research lab, we are dedicated to unraveling the underlying mechanisms that contribute to the development and progression of diabetes. Our primary focus lies in investigating factors and gene regulatory networks that have the potential to safeguard pancreatic islets from damage during the advanced stages of this condition. Through our findings and ongoing research, we strive to make significant contributions to the development of more effective treatments and personalized approaches that can enhance the management of diabetes.
Our ultimate goal is to improve the lives of individuals affected by diabetes.
</p>

# Teaching {#teaching}  
<p style="text-align: justify;">
As an experienced educator, I am committed to providing my students with a comprehensive, conceptual, and practical understanding of the course materials. My teaching philosophy revolves around the idea that rigorous teaching and training can transform the younger generations into highly skilled citizens capable of addressing local and global challenges through motivation, innovation, and scientific approach. I have been teaching following courses at the College of Science, University of Jeddah.   
</p>

### M.Sc. Courses
- Computer Application in Biochemistry (BIOC-617)
- Immunobiochemistry (BIOC-653)
- Microbial Bioinformatics and Biological Databases (BIO-602)
- Forensic Bioinformatics (BIOC-631)
- Forensic Computing (BIOC-606)


### B.Sc. Courses
- Computer Application in Biochemistry (SCBC-431)
- Cancer Biochemistry (SCBC-422)
- Immunobiochemistry (SCB-433)
- Biotechnology and Analytical Methods (SCBC-312)
- Laboratories safety and security (SCBC-211)

# Publications (selected) {#publication}


<div style="padding-left: 30px;">
<p style="text-align: justify;">

1: F. Ahmed. [Deciphering the gene regulatory network associated with anti-apoptosis in the pancreatic islets of type 2 diabetes mice using computational approaches](http://dx.doi.org/10.3934/bioeng.2023009). <i>AIMS Bioengineering</i> 2023.

2: E. Ahmad. [Molecular approaches in cancer](https://www.ncbi.nlm.nih.gov/pubmed/36244434). <i>Clin Chim Acta</i> 2022.

3: F. Ahmed. [A Systems Biology and LASSO-Based Approach to Decipher the Transcriptomeâ€“Interactome Signature for Predicting Non-Small Cell Lung Cancer](https://www.mdpi.com/2079-7737/11/12/1752). <i>Biology</i> 2022.

4: N. A. Baeshen. [In silico screening of some compounds derived from the desert medicinal plant Rhazya stricta for the potential treatment of COVID-19](https://www.ncbi.nlm.nih.gov/pubmed/35778482). <i>Sci Rep</i> 2022.

5: M. R. Mohammed. [The Histone H3K27me3 Demethylases KDM6A/B Resist Anoikis and Transcriptionally Regulate Stemness-Related Genes](https://www.frontiersin.org/article/10.3389/fcell.2022.780176). <i>Frontiers in Cell and Developmental Biology</i> 2022.

6: M. Rehan. [A Computational Approach Identified Andrographolide as a Potential Drug for Suppressing COVID-19-Induced Cytokine Storm](https://www.ncbi.nlm.nih.gov/pubmed/34248936). <i>Front Immunol</i> 2021.

7: F Ahmed. [pssRNAit: A Web Server for Designing Effective and Specific Plant siRNAs with Genome-Wide Off-Target Assessment](https://doi.org/10.1104/pp.20.00293). <i>Plant Physiology</i> 2020.

8: F. Ahmed. [A Network-Based Analysis Reveals the Mechanism Underlying Vitamin D in Suppressing Cytokine Storm and Virus in SARS-CoV-2 Infection](https://www.frontiersin.org/article/10.3389/fimmu.2020.590459). <i>Frontiers in Immunology</i> 2020.

9: F. Ahmed. [A Comprehensive Analysis of cis-Acting RNA Elements in the SARS-CoV-2 Genome by a Bioinformatics Approach](https://www.frontiersin.org/article/10.3389/fgene.2020.572702). <i>Frontiers in Genetics</i> 2020.

10: F. A. Alzahrani. [Investigating the pathogenic SNPs in BLM helicase and their biological consequences by computational approach](https://www.ncbi.nlm.nih.gov/pubmed/32704157). <i>Sci Rep</i> 2020.

11: F. Ahmed. [Integrated Network Analysis Reveals FOXM1 and MYBL2 as Key Regulators of Cell Proliferation in Non-small Cell Lung Cancer](https://www.ncbi.nlm.nih.gov/pubmed/31681566). <i>Front Oncol</i> 2019.

12: H. Choudhry. [Middle East respiratory syndrome: pathogenesis and therapeutic developments](https://www.ncbi.nlm.nih.gov/pubmed/32201499). <i>Future Virol</i> 2019.

13: F. Ahmed. [Bioinformatics tools for achieving better gene silencing in plants](https://www.ncbi.nlm.nih.gov/pubmed/25740355). <i>Methods Mol Biol</i> 2015.

14: F. Ahmed. [A molecular bridge: connecting type 2 diabetes and Alzheimer's disease](https://www.ncbi.nlm.nih.gov/pubmed/24059325). <i>CNS Neurol Disord Drug Targets</i> 2014.

15: F. Ahmed. [Comprehensive analysis of small RNA-seq data reveals that combination of miRNA with its isomiRs increase the accuracy of target prediction in Arabidopsis thaliana](https://www.ncbi.nlm.nih.gov/pubmed/25629686). <i>RNA Biol</i> 2014.

16: F. Ahmed. [PHDcleav: a SVM based method for predicting human Dicer cleavage sites using sequence and secondary structure of miRNA precursors](https://www.ncbi.nlm.nih.gov/pubmed/24267009). <i>BMC Bioinformatics</i> 2013.

17: F. Ahmed. [Mining Functional Elements in Messenger RNAs: Overview, Challenges, and Perspectives](https://www.ncbi.nlm.nih.gov/pubmed/22639614). <i>Front Plant Sci</i> 2011.

18: F. Ahmed. [Designing of highly effective complementary and mismatch siRNAs for silencing a gene](https://www.ncbi.nlm.nih.gov/pubmed/21853133). <i>PLoS One</i> 2011.

19: A. Tyagi. [HIVsirDB: a database of HIV inhibiting siRNAs](https://www.ncbi.nlm.nih.gov/pubmed/22022467). <i>PLoS One</i> 2011.

20: F. Ahmed. [Prediction of guide strand of microRNAs from its sequence and secondary structure](https://www.ncbi.nlm.nih.gov/pubmed/19358699). <i>BMC Bioinformatics</i> 2009.

21: F. Ahmed. [Prediction of polyadenylation signals in human DNA sequences using nucleotide frequencies](https://www.ncbi.nlm.nih.gov/pubmed/19795571). <i>In Silico Biol</i> 2009.
</p>
</div>

# Bioinformatics Tools {#tools}

-	[NSCLCpred](https://github.com/firozimtech/NSCLCpred): An online tool for predicting non-small cell lung cancer. [Webserver](https://hifzuransari.shinyapps.io/NSCLCpred/)    

-	[pssRNAit](http://plantgrn.noble.org/pssRNAit/): A server based tool for designing effective and specific siRNA in animals and plants.

- [PolyApred](www.imtech.res.in/raghava/polyapred/): A server based tool for predicting polyadenylation signals in human DNA sequence.

- [RISCbinder](http://crdd.osdd.net:8081/RISCbinder/): A server based tool for predicting guide strand of miRNA among the miRNA:miRNA* duplex.

- [PHDcleav](www.imtech.res.in/raghava/phdcleav/): A server based tool for predicting human Dicer cleavage sites in pre-miRNA.

- [desiRm](www.imtech.res.in/raghava/desirm/): A server based tool for designing siRNA of desired efficacy.

- [GPSR package](www.imtech.res.in/raghava/gpsr): A comprehensive software package for genomics, proteomics and system biology. (Contributed in the development of the package.)

- [HIVsirDB](http://crdd.osdd.net/raghava/hivsir/index.php): An online database of HIV inhibiting siRNAs.

- CelRNAiMapper: In-house pipeline and database for RNA interference clone mapping in Caenorhabditis elegans.



# Team and Collaborators {#team}

<div style="padding-left: 30px;">
## Lab Members
### Current
#### M.Sc.
- Abdullah Sultan Alotaibi    

### Alumni
#### M.Sc.   
- Ahmad Mohammad Alghamdi
- Abdulla Grosh Almaliki
- Mohesn Ali Alkahtani
- Hassan Ali Alzahrani
- Naif Muneer Alharbi

#### B.Sc. 
- Abdullah Khalid 
- Anas Abdullah 
- Faisal Abdullah
- Mohammad Ahmad Qarni 
- Motaz Khalid Yamni 
- Fahad Faham Taybi
- Mohammed Jari Alshmrani
- Mohammed Anwar Abdul Rahim Sadat 
- Aws Mounir Hammoud Al-Buqami 
- Majid Ali Al-Qarni
- Abdulsalam Abdullah Saeed Al Ghamdi
- Abdulaziz Musa RajaAllah Al-Salami
- Sultan Muhammad Ahmad Al-Yami





## Collaborators
- Dr. N.K.  Mishra, St. Jude Children Research Hospital, USA
- Dr. M.S. Jamal, Wayn State University, USA
- Dr. A. Haque, King Abdulaziz University, Saudi Arabia
- Dr. M. Rehan, King Abdulaziz University, Saudi Arabia
- Dr. S. S. Sohrab, King Abdulaziz University, Saudi Arabia
- Dr. M. I. Khan, King Abdulaziz University, Saudi Arabia
- Prof. M.N. Baeshen, University of Jeddah, Saudi Arabia
- Dr. M.W. Khan, University of Jeddah, Saudi Arabia
- Dr. E. Ahmad, G. B. Pant Institute of Postgraduate Medical Education and Research, India 
- Dr. A. A. Khan, National AIDS Research Institute, Pune, India
- Dr. H.R. Ansari, King Abdullah International Medical Research Center, Jeddah, Saudi Arabia
</div>

# News {#news}
<p style="text-align: justify;">
Explore our captivating research story, featured in popular magazines and news outlets, revealing intriguing details and exciting discoveries that have captured the media's attention.    
</p>

- [_pssRNAit_](https://www.outlookindia.com/website/story/feature-crop-research/361966/): Embark on a scientific journey with pssRNAit, our revolutionary bioinformatics tool. Harness pssRNAit's power to design precise RNA sequences that silence specific genes to uncover plant secrets and reshape crop development. Experience the transformative impact of our software, creating superior crop varieties to feed a growing population.

- [_Plant Physiology_](https://plantae.org/recognizing-plant-physiology-authors-firoz-ahmed/), a high-impact journal by American Society of Plant Biologists, recognizes Dr. Ahmed's contributions as the first author of a breakthrough technology, highlighting his significant role in advancing the field of gene silencing.

- Unveiling Molecular Insights: Vitamin D's role in suppressing cytokine storm and fighting SARS-CoV-2 infection. ([in Hindi](https://www.shrinaradmedia.com/indian-scientist-discovers-mechanism-behind-vitamin-d-benefit-in-covid-19/?fbclid=IwAR1DrE19-boOcZIC42toEjYDYdJrAxzosLAUhsr8BtH73kL28BIqfO8o_AA))





# Contact {#contact}
__For any inquiries about collaboration opportunities please donâ€™t hesitate to contact us:__ 

- __Dr. Firoz Ahmed__  
  Assistant Professor (Bioinformatics and Systems Biology)  
  Department of Biochemistry, College of Science  
  University of Jeddah, Jeddah, Saudi Arabia  
  Email: fahmed1@uj.edu.sa
  



<footer class="footer">
  &copy; 2023 Ahmed Lab. All rights reserved. 

    <a href="https://www.linkedin.com/in/firoz-ahmed-b3573914/" target="_blank" rel="noopener noreferrer"><img src="images/linkedin_logo.jpeg" alt="LinkedIn" width="90" height="30"></a>
    <a href="https://scholar.google.com/citations?user=WCCGWG0AAAAJ&hl=en/" target="_blank" rel="noopener noreferrer"><img src="images/scholar_logo.jpeg" alt="Google Scholar" width="100" height="40"></a>
 

![Visitors](https://visitor-badge.glitch.me/badge?page_id=firozimtech.firozimtech.github.io)


[![Visitors](https://shields.io/badge/dynamic/json?label=Visitors&query=value&url=https%3A%2F%2Fapi.countapi.xyz%2Fhit%2Ffirozimtech.firozimtech.github.io%2Fcount-badge&style=for-the-badge)](https://github.com/firozimtech/firozimtech.github.io)


</footer>
